nilearn/glm/first_level/first_level.py

```diff
@@ -1321,6 +1323,169 @@ def _prepare_mask(self, run_img):
 
         self.n_elements_ = self.masker_.n_elements_
 
+    def _plotting_pred_and_res(
+        self,
```

Comment:
self is never used, meaning that this should probably be a stand-alone function rather than a class method.

Reply:
I think it's pretty much tied to the GLM anyway.